src/orca/Orca.java

```diff
@@ -0,0 +1,267 @@
+package orca;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import java.util.Scanner;
+
+public class Orca {
+    static final String LOGO = "     _______  ______    _______  _______ \n"
+            + "    |       ||    _ |  |       ||   _   | \n"
+            + "    |   _   ||   | ||  |       ||  |_|  | \n"
+            + "    |  | |  ||   |_||_ |       ||       | \n"
+            + "    |  |_|  ||    __  ||      _||       | \n"
+            + "    |       ||   |  | ||     |_ |   _   | \n"
+            + "    |_______||___|  |_||_______||__| |__| \n";
+    static final boolean FINISHED = true;
+    static final String FILE_PATH = "./data/orca.txt";
+
+    static ArrayList<Task> tasks = new ArrayList<>();
+    static Scanner in = new Scanner(System.in);
+    static String userInput = "";
+    static CommandType commandType;
+    static int taskNo;
+    static Task newTask;
+
+    public Orca() {
+        File f = new File(FILE_PATH);
+        if (!f.exists()) {
+            f.getParentFile().mkdirs();
+        } else {
```

Comment:
Try to avoid deep nesting / arrowhead style code as it is difficult to read and debug in the future.